skyportal/app_server.py

```diff
@@ -326,6 +327,11 @@ def log_request(self, handler):
     (r'/api/user(/[0-9]+)/roles(/.*)?', UserRoleHandler),
     (r'/api/user(/.*)?', UserHandler),
     (r'/api/weather(/.*)?', WeatherHandler),
+    # strictly require uuid4 token for this unauthenticated endpoint
+    (
+        r'/api/webhook/(obj)_analysis/([0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12})?',
```

Comment:
This 89ab is part of uuid4 specifically?

Reply:
Yes, as is the 4 before [0-9a-f]{3}.